gst/nnstreamer/elements/gsttensor_transform.c

```diff
@@ -1989,12 +1989,27 @@ gst_tensor_transform_transform_caps (GstBaseTransform * trans,
 
   if (filtercap && gst_caps_get_size (filtercap) > 0) {
     GstCaps *intersection;
+    GstPad *pad;
+    GstCaps *peer_caps;
+
+    gst_tensor_caps_update_dimension (result, filtercap);
 
     intersection =
         gst_caps_intersect_full (result, filtercap, GST_CAPS_INTERSECT_FIRST);
 
     gst_caps_unref (result);
     result = intersection;
+
+    if (direction == GST_PAD_SINK)
+      pad = GST_BASE_TRANSFORM_SRC_PAD (filter);
+    else
+      pad = GST_BASE_TRANSFORM_SINK_PAD (filter);
+
+    if ((peer_caps = gst_pad_peer_query_caps (pad, NULL))) {
+      structure = gst_caps_get_structure (peer_caps, 0);
```

Comment:
structure is not used. so can we remove this ?

Reply:
Removed not used structure
Thank you :)